ci/aws-analytical-env/shared/meta.yml

```diff
@@ -50,6 +50,9 @@ meta:
           args:
             - -exc
             - |
+              apk add -q --update --no-cache python3 && ln -sf python3 /usr/bin/python
```

Comment:
Why are you installing python? None of the things we are executing use python

Reply:
@IanUtley - I think it is for the "data_cognito_user_roles.py" script?

Reply:
Ok, its because we are using external datasource.